products/alp/main.pm

```diff
@@ -18,7 +18,15 @@ my $distri = testapi::get_required_var('CASEDIR') . '/lib/susedistribution.pm';
 require $distri;
 testapi::set_distribution(DistributionProvider->provide());
 
-return 1 if load_yaml_schedule;
+# Containerized YaST needs proper libyui initialization
+if (load_yaml_schedule) {
+    if (YuiRestClient::is_libyui_rest_api) {
+        YuiRestClient::set_libyui_backend_vars;
```

Comment:
Why is this block needed here? Can't you just use `YAML_SCHEDULE` variable in the jobs?